mappings/net/minecraft/realms/RealmsSerializable.mapping

```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/class_4885 net/minecraft/realms/RealmsSerializable
+	COMMENT A marker interface for Gson serializable pojos; the implementers make
+	COMMENT sure that they have {@code SerializedName} annotation on all their
+	COMMENT data fields so serialization works after obfuscation, and save/load of
+	COMMENT such objects are controlled through another serializer.
+	COMMENT
+	COMMENT <p>This is a new class in 20w10a.
```

Comment:
```suggestion
	COMMENT @since 20w10a.
```

Reply:
Im not sure we really need this info here. In theory you can get it from using the intermediary names.

Reply:
Not really; I add this because this is not a renamed class or merged class (i.e. proxy merges) in 10a, but rather a completely new class

Reply:
It's still unneeded info

Reply:
So consider class_4900 and class_4885: 4900 is created because matcher failed to match 4900 with old Realms class (or the class is just recently obfuscated). It existed before; however, 4885 is totally new, like PiglinEntity or PiglinBrain classes. This javadoc is peripheral and may be changed side-effect-free in the future, shouldn't be a big problem if this `@since` is one (clearly it isn't)